roles/app/tasks/main.yml

```diff
@@ -227,6 +227,7 @@
 
 - name: log dir
   file: path="{{ nginx_logs }}" state=directory owner=www-data group=adm
+  when: app == 1
```

Comment:
nginx logs looks more like http than app

Reply:
Agree
Fixed in squashed commit